zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/ParagraphTest.java

```diff
@@ -70,6 +70,18 @@ public void scriptBodyWithReplName() {
 
     text = "%table 1234567";
     assertEquals("1234567", Paragraph.getScriptBody(text));
+
+    text = "%presto (testPropertyKey) 1234567";
+    assertEquals("(testPropertyKey) 1234567", Paragraph.getScriptBody(text));
+
+    text = "%spark (1234567)";
+    assertEquals("(1234567)", Paragraph.getScriptBody(text));
+
+    text = "%jdbc(presto) (testPropertyKey) 1234567";
+    assertEquals("(testPropertyKey) 1234567", Paragraph.getScriptBody(text));
+
+    text = "%jdbc(presto) (testPropertyKey) (select 1234567)";
+    assertEquals("(testPropertyKey) (select 1234567)", Paragraph.getScriptBody(text));
```

Comment:
IIUC, the scriptBody should be `(presto) (testPropertyKey) (select 1234567)` which would be passed to JdbcInterpreter. Otherwise JdbcInterpreter don't know this is `presto`. \cc @prabhjyotsingh